Rakefile

```diff
@@ -1,6 +1,7 @@
 require "yast/rake"
 
 Yast::Tasks.submit_to :casp10
+obs_project = "Devel:YaST:CASP:1.0"
```

Comment:
To make it build in Travis. lslezak found out yesterday that this helps (at least for some of those four packages).

But there was no code change anyway; Travis is just actively obstructing my work.